Makefile

```diff
@@ -45,7 +45,7 @@ dev:
 .PHONY: dev-all
 dev-all:
 	$(eval LD_FLAGS_RUN = "-w -X $(REPOSITORY)/pkg/version.Version="$(shell ./hack/get-next-version))
-	@KUBECONFIG=dev/garden-kubeconfig.yaml go run -ldflags $(LD_FLAGS_RUN) cmd/garden-controller-manager/main.go --config=dev/componentconfig-garden-controller-manager.yaml
+	@go run -ldflags $(LD_FLAGS_RUN) cmd/garden-controller-manager/main.go --config=dev/componentconfig-garden-controller-manager.yaml
```

Comment:
If we remove this we need to document somewhere that the `clientConnection.kubeconfig` field in the Garden controller manager's Componentconfig must be set properly.